ql/time/period.cpp

```diff
@@ -262,7 +257,8 @@ namespace QuantLib {
         std::pair<Integer,Integer> daysMinMax(const Period& p) {
             switch (p.units()) {
               case Days:
-                return std::make_pair(p.length(), p.length());
+                // Approximately 1.4 calendar days in a business day on average.
```

Comment:
I'm not sure I would change this as well&mdash;it doesn't cause an error like `normalize`, and whoever calls this is probably aware of the context.

In the long term I'd go with `Period(5, CalendarDays)` and `Period(5, BusinessDays)` doing the correct thing and with `Days` being phased out, but it will take quite some work.

Reply:
I initially thought that this change is needed to stop operator== from returning true for `7*Days == 1*Weeks`, but I just checked now and it does not make a lot of difference. Reverted.

I agree that ultimately, CalendarDays and BusinessDays should be separate units.